src/os/unix.rs

```diff
@@ -225,6 +225,13 @@ pub trait WindowBuilderExt {
     fn with_resize_increments(self, increments: LogicalSize) -> WindowBuilder;
     /// Build window with base size hint. Only implemented on X11.
     fn with_base_size(self, base_size: LogicalSize) -> WindowBuilder;
+
+    /// Build window with a given app-id. It should match the `.desktop`
+    /// file distributed with your program. Only relevant on Wayland.
+    ///
+    /// For details about application id conventions, see the
```

Comment:
```suggestion
    /// For details about application ID conventions, see the
```
To be more consistent with the stylization you used in the CHANGELOG